charts/cluster-autoscaler/Chart.yaml

```diff
@@ -11,4 +11,4 @@ name: cluster-autoscaler
 sources:
   - https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
 type: application
-version: 9.37.0
+version: 9.37.1
```

Comment:
@blanchardma, As this is a new feature request, please bump this into a minor version, not a patch version.

Reply:
I only bump the patch version because the improvements are commented out and therefore not active, so this is more of a recommendation. But I can adapt it if you stick with it.

Reply:
@blanchardma, Hi, we update the patch version when there is some bug, but you mention this PR as a feature so I think it should be a minor bump version. So that was my only concern, other than that changes look good to me.

Thanks!

Reply:
@Shubham82 I fixed the version.

Reply:
I'd agree with @blanchardma here, it's an example documentation change only, we're not changing the functionality exposed at all, so a patch release increment makes more sense.

Reply:
> I'd agree with @blanchardma here, it's an example documentation change only, we're not changing the functionality exposed at all, so a patch release increment makes more sense.

Thanks, @gjtempleton for clarification.